infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -236,7 +236,7 @@ public TokenType txnCanCommit(TxResolutionInfo txInfo, /** Input. */
                 // for each key pair, check for conflict;
                 // if not present, check against the wildcard
                 conflictParamSet.forEach(conflictParam -> {
-                    int conflictKeyHash = getConflictHashCode(entry.getKey(),
+                    String conflictKeyHash = getConflictHashCode(entry.getKey(),
```

Comment:
ok maybe I did not understand the test case correctly. In my conversation with Anny,
she mentioned that the hash for 2 UUIDs (keys) collided and so the streamId did not matter in the scheme. We should check with her once.
@annym could you please provide input.